src/contributors/shelb-doc.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "Shelby De Oliveira"
+username: "shelb-doc"
+image: "shelb-doc.jpg"
+facebook: "https://www.facebook.com/ReportTurret/"
```

Comment:
Please just provide the username instead full url. LIKE `facebook: ReportTurret`. Same like this change other usernames too.

Reply:
fixed